sdk/core/azure-core/inc/azure/core/etag.hpp

```diff
@@ -87,7 +87,7 @@ class ETag final {
       return !left.m_value && !right.m_value;
     }
 
-    switch (comparisonKind)
+    switch (comparisonKind) // LCOV_EXCL_LINE
```

Comment:
If we had `default: _azure_UNREACHABLE_CODE()`, could we get rid of this comment?

Reply:
Gcovr on this line won't be satisfied that we don't pass a value that brings us to `default` label, even though the label itself is excluded.